src/main/java/com/hotels/bdp/circustrain/bigquery/conversion/BigQueryToHiveTypeConverter.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.conversion;
+
+public class BigQueryToHiveTypeConverter {
+  public String convert(String type) {
+    type = type.toUpperCase();
+    if ("STRING".equals(type)) {
```

Comment:
This would look better as a switch statement as you can then group items which map to the same type together.